src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/NoCloneCheck.java

```diff
@@ -79,42 +81,57 @@
  * force the calling client to handle a CloneNotSupportedException.  They also
  * are typed therefore no casting is necessary. Finally, they are more
  * flexible since they can take interface types rather than concrete classes.
- *
+ * </p>
  * <p>Sometimes a copy constructor or static factory is not an acceptable
  * alternative to the clone method.  The example below highlights the
  * limitation of a copy constructor (or static factory). Assume
  * Square is a subclass for Shape.
- *
+ * </p>
  * <pre>
  * Shape s1 = new Square();
  * System.out.println(s1 instanceof Square); //true
  * </pre>
+ * <p>
  * ...assume at this point the code knows nothing of s1 being a Square
  *    that's the beauty of polymorphism but the code wants to copy
  *    the Square which is declared as a Shape, its super type...
- *
+ * </p>
  * <pre>
  * Shape s2 = new Shape(s1); //using the copy constructor
  * System.out.println(s2 instanceof Square); //false
  * </pre>
+ * <p>
  * The working solution (without knowing about all subclasses and doing many
  * casts) is to do the following (assuming correct clone implementation).
- *
+ * </p>
  * <pre>
  * Shape s2 = s1.clone();
  * System.out.println(s2 instanceof Square); //true
  * </pre>
+ * <p>
  * Just keep in mind if this type of polymorphic cloning is required
  * then a properly implemented clone method may be the best choice.
- *
+ * </p>
  * <p>Much of this information was taken from Effective Java:
  * Programming Language Guide First Edition by Joshua Bloch
  * pages 45-52.  Give Bloch credit for writing an excellent book.
  * </p>
+ * <p>
+ * This check is almost exactly the same as the {@code NoFinalizerCheck}.
```

Comment:
Should this and the `See` go before the raionale?

Reply:
done